doc/lua_api.txt

```diff
@@ -7885,6 +7885,30 @@ Used by `minetest.register_node`.
         -- Default: nil
         -- Warning: making a liquid node 'floodable' will cause problems.
 
+        on_fall = function(pos, node)
+        -- Called before node starts to fall.
+        -- If a boolean is returned, the builtin falling node
+        -- handling is skipped. `true` denotes successful falling.
+        -- By default, the node is removed and a falling node
+        -- entity is spawned at `pos`.
+        -- default: nil
```

Comment:
- There should be more information on what successful falling means (update spread to neighbours).
- The "builtin" falling node handling should probably also be called "default". Otherwise one could understand the "By default" in the last sentence as "If this callback is not defined (`on_fall = nil`)".
- The default / builtin falling node handling includes calling the `after_fall` callback. Imo that's an important detail to be mentioned here.